src/circuit/gadget/poseidon/pow5t3.rs

```diff
@@ -110,10 +110,14 @@ impl<F: FieldExt> Pow5T3Chip<F> {
                         - next[next_idx].clone())
             };
 
-            vec![full_round(0), full_round(1), full_round(2)]
+            vec![
+                ("state[0]", full_round(0)),
+                ("state[1]", full_round(1)),
+                ("state[2]", full_round(2)),
```

Comment:
```suggestion
                ("state[0] full round", full_round(0)),
                ("state[1] full round", full_round(1)),
                ("state[2] full round", full_round(2)),
```

Reply:
The gate is named "full round", so this is duplicative and unhelpful to debugging.